frontend/src/exploration/previews/LayoutCard.tsx

```diff
@@ -0,0 +1,51 @@
+import {
+  Card,
+  CardContent,
+  CardMedia,
+  ListItem,
+  makeStyles,
+  Typography,
+} from '@material-ui/core';
+import React from 'react';
+import { Link } from 'react-router-dom';
+import { createStyles } from '@material-ui/core/styles';
+import LayoutDefinition from './LayoutDefinition';
+
+const useStyle = makeStyles(() =>
+  createStyles({
+    card: {
+      width: '100%',
+    },
+  })
+);
+
+/**
+ * A Preview of a Layout that routes to the tab path from the input parameter.
+ * @param layout - Data used to construct the card
+ */
+function LayoutCard(layout: LayoutDefinition): JSX.Element {
```

Comment:
[Optional]
Better (imo) semantic naming: It's not for layouts, it's for visualisations. If you rename it then apply it to the others as well as
- LayoutCard => VisualisationPreviewCard
- LayoutDefinition => VisualisationPreviewDefinition
- layoutsData => visualisationPreviewData

```suggestion
/**
 * A preview of a visualisation that routes to the tab path from the input parameter.
 * @param visualisation - Data used to construct the card
 */
function VisualisationPreviewCard(visualisation: VisualisationPreviewDefinition): JSX.Element {
```